src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py

```diff
@@ -1492,7 +1492,7 @@ def load_annulus_description(
         "pvt_model": load_value,
         "top_node": load_value,
         "initial_conditions": load_initial_conditions_description,
-        "gas_lift_valve_equipment": load_gas_lift_valve_equipment_description,
+        "equipment": load_annulus_equipment_description,
```

Comment:
Did you test the auto loaders?

Reply:
by auto loaders do you mean these loaders (load_annulus_equipment_...)? If so some tests already cover it

Reply:
See https://github.com/ESSS/alfasim-sdk/pull/210#discussion_r760541697

> Maybe using directly `load_instance`.